pennylane/kernels/__init__.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+This subpackage defines functions that relate to quantum kernel methods.
+On one hand this includes functions to call a quantum kernel systematically
+on training and test datasets to obtain the *kernel matrix*.
+On the other hand it provides postprocessing methods for those kernel
+matrices which can be used to mitigate device noise and sampling errors.
+
+Given a kernel
+
+.. math ::
+
+    k: \mathbb{R}^d \times \mathbb{R}^d &\to \mathbb{R}, \
+    (x_1, x_2)&\mapsto k(x_1, x_2)
```

Comment:
Unfortunately this is not rendering: 
![image](https://user-images.githubusercontent.com/2959003/118839859-72f5a200-b8f9-11eb-89f4-cdcc9f9f99a8.png)

(see: https://pennylane--1102.org.readthedocs.build/en/1102/code/qml_kernels.html)

Reply:
That should be fixed now

Reply:
I think it does not quite work yet. Did you prefer to put it in one line?

Reply:
Yes, it looked a bit weird because it was not aligned. I thought \ or \quad would be safe for spaces, but it's always a bit hard because the feedback loop is kind'a slow if you always have to wait for the docs compilation job (I know I could build it locally, but making that work usually is also quite demanding 😅

Reply:
:D I think I figured it out: The & were trying to align the one-liner now, which does not work
Should be fixed :)